test/installer.js

```diff
@@ -63,9 +64,37 @@ describe('module', function () {
         }
       }, done)
     })
+  })
+
+  describe('with an app without a homepage or author URL', function (test) {
+    var baseDir = path.join(os.tmpdir(), 'electron-installer-redhat', 'app-without-homepage')
+    var dest = 'test/fixtures/out/baz/'
+
+    before(function (done) {
+      if (fs.existsSync(baseDir)) {
+        fs.removeSync(baseDir)
+      }
+      fs.copySync('test/fixtures/app-without-asar', baseDir)
+      var packageJSONFilename = path.join(baseDir, 'resources', 'app', 'package.json')
+      var packageJSON = JSON.parse(fs.readFileSync(packageJSONFilename))
+      packageJSON.author = 'Test Author'
+      fs.writeFileSync(packageJSONFilename, JSON.stringify(packageJSON))
```

Comment:
Yeah. I still find that a very odd thing that you can do with `require()`, though.